.github/workflows/ci-config.yml

```
@@ -0,0 +1,34 @@
+# Workflow for processing and publishing bikeshed proposals to github pages
+
+name: verify proposals
+on:
+  push:
+    # only concerned with changes in the master branch
+    branches: [ master ]
+    # only invoke when a bikeshed document is updated
+    # paths: [ '*.bs' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: upgrade pip
+        run: python3 -m pip install --upgrade pip
+
+      - name: install bikeshed
+        run: |
+          python3 -m pip install --upgrade bikeshed
+          bikeshed update
+
+      - name: generate html
+        run: for index in proposals/*/index.bs; do bikeshed spec $index; done
```

We could do something a little bit more generic in case we want to reuse the action elsewhere, but it's not even really necessary.

All looks good to me!

```suggestion
        run: for bsdoc in proposals/**/*.bs; do bikeshed spec $bsdoc; done
```